jcl/jpp_configuration.xml

```diff
@@ -241,6 +241,34 @@
 		<parameter name="jxerules:outputdir" value="java/lang"/>
 	</configuration>
 
+	<configuration
+		  label="JAVA25"
+		  outputpath="JAVA25/src"
+		  dependencies="JAVA24"
+		  jdkcompliance="22">
```

Comment:
The latest version of eclipse supports Java 23: we should use `jdkcompliance="23"` for JAVA23-25.

Reply:
Updated